SU-HSDO/suhumsci
# SHS-5011: Testimonial component should use blockquote/cite markup

# READY FOR REVIEW

## Summary
- Use `<blockquote>` instead of `<div>` in testimonial component

## Need Review By (Date)
- 09/01

## Urgency
- medium

## Steps to Test
- Create a Flexible page and add some testimonials (or visit `/testimonials` if you're in traditional or colorful sites)
- Confirm that there are not visual changes in the components
- Inspect the source code and confirm that the testimonial quote is in a `<blockquote>` tag.

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)
